docs/src/Checkbox.doc.js

```diff
@@ -144,6 +144,33 @@ class CheckboxExample extends React.Component {
   />
```

Comment:
@mminute could we update the flowtypes in the docs?

![image](https://user-images.githubusercontent.com/127199/43112478-2eaa841a-8eab-11e8-93f4-6a7c27327032.png)